cs/API-v2/Program.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Security.Cryptography;
+using System.Text;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.OpenSsl;
+using Org.BouncyCastle.Security;
+
+namespace WalutomatApiV2Example
+{
+    class Program
+    {
+        static string host = "https://api.walutomat.pl";
+        static void Main(string[] args)
+        {
+            switch (args[0])
+            {
+                case "orders-active":
+                    getActiveOrders();
+                    break;
+                case "wallet":

```

Comment:
`"wallet"` → `"wallet-balance"`?

Reply:
makes sense

Reply:
you need to update this in README.md as well